config/qualifications.yml

```diff
@@ -40,20 +44,70 @@ description: >
   - In section 1.5.1, there are instructions to set up a Heroku account to deploy your code to.
     Since you'll be deploying code to production in a few short months, you can skip all related steps.
 ---
-title: 'Complete a ruby tutorial'
+title: Complete a ruby tutorial
```

Comment:
Shouldn't ruby come before rails?

Reply:
"That's debatable" Matt said, and I agree. You don't have to know Ruby to learn Rails, in fact, and I think becoming proficient at Ruby first could be confusing. 

That being said, it doesn't really make sense to try to do most of this file in order, just the first couple bits. So in reality they can do whatever.